lib/code_corps/helpers/query.ex

```diff
@@ -9,7 +9,7 @@ defmodule CodeCorps.Helpers.Query do
 
   # skill queries
 
-  def limit_filter(query, %{"limit" => count}) do
+  def limit_filter(query, %{"limit" => count}) when is_integer(count) do
```

Comment:
Seems like query might be serializing this as a string, actually, so tests are failing. Either that, or our tests are sending a string `"5"`, when they should be sending an integer.